main.cpp

```diff
@@ -1,13 +1,14 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2019 ARM Limited
+ * SPDX-License-Identifier: Apache-2.0
+ */
 #include "mbed.h"
 #include "LittleFileSystem.h"
 #include "SDBlockDevice.h"
 #include <stdio.h>
 #include <errno.h>
-/* mbed_retarget.h is included after errno.h so symbols are mapped to
```

Comment:
this is in the commit for license addition, how is it related? why it was removed?

Reply:
@0xc0170 i wasn't sure if the commit was relevant seeing as it was added some years ago. Thought it was trivial to clean it up here. Would you recommend I revert the change or move it into another commit?

Reply:
own commit +1